seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -294,7 +294,9 @@
         "function countToFive() {",
         "  let firstFive = \"12345\";",
         "  let len = firstFive.length;",
+        "  //Fix this line",
```

Comment:
Nitpicking, but can you put a space in front of `Fix`? Most of the comments I've seen in the seed have a space between the `//` and the comments themselves.